cfme/tests/services/test_iso_service_catalogs.py

```diff
@@ -93,7 +96,28 @@ def catalog_item(appliance, provider, dialog, catalog, provisioning):
     )
 
 
+# There is a Libvirt bug BZ(1783355) on our RHV 4.4. This seem to prevent some tests to run with it.
```

Comment:
@jaryn interesting approach here, but I think we would be fine using this BZ as a blocker directly.

Yes there is a separation from when that BZ is unblocked to when our RHV4.4 provider is updated with the fix, but this test going from skipped to failing is a good indicator that its time to update our provider to pickup the BZ fix.